experimental/SetPartitions/README.md

```diff
@@ -0,0 +1,4 @@
+Implementation of set-partitions as used for easy quantum groups, see for example:
+* Section 1.2 & 1.3 of https://arxiv.org/abs/1312.3857
+* Section 1 of https://arxiv.org/abs/1509.00988
+* Section 2 of https://arxiv.org/abs/1609.02321
```

Comment:
Please follow the format set by `experimental/ExperimentalTemplate/README.md`. Also, hyperlinks in Markdown should be surrounded by `<` and `>`, i.e.
```suggestion
* Section 1.2 & 1.3 of <https://arxiv.org/abs/1312.3857>
* Section 1 of <https://arxiv.org/abs/1509.00988>
* Section 2 of <https://arxiv.org/abs/1609.02321>
```

Reply:
Also I think a little more explanation should be in here, in particular, that this is *not* what ordinary mathematicians would think of when talking about partitions